src/main/scala/au/csiro/variantspark/cli/ImportanceCmd.scala

```diff
@@ -124,6 +133,8 @@ class ImportanceCmd extends ArgsApp with SparkApp
   
     val inputData = DefTreeRepresentationFactory.createRepresentation(featureSource.features.zipWithIndex()).cache() 
     val totalVariables = inputData.count()
+    echo(s"Input data: ${inputData}")
```

Comment:
Why is this needed?

Reply:
I must have missed this. Removing it now.